setup.py

```diff
@@ -46,10 +46,6 @@
 	sourceFileWithVersion=packageInformationFile,
 	developmentStatus="beta",
 	classifiers=[
```

Comment:
The behavior of `pyTooling.Packaging` is not yet changed to merge classifiers. I also think it's not a wise solution to enforce e.g. audience developers for every package released with pyTooling.Packaging.

Reply:
Do you mean that we should not define `classifiers` in the EDAA repos?

Reply:
Discussed in a meeting. We both meant the same: using the defaults explicitly here, and appending additional classifiers.